source/Energinet.DataHub.MarketParticipant.Application/Handlers/Actor/UpdateActorHandler.cs

```diff
@@ -45,7 +46,8 @@ public sealed class UpdateActorHandler : IRequestHandler<UpdateActorCommand>
             IActorIntegrationEventsQueueService actorIntegrationEventsQueueService,
             IOverlappingBusinessRolesRuleService overlappingBusinessRolesRuleService,
             IAllowedGridAreasRuleService allowedGridAreasRuleService,
-            IExternalActorIdConfigurationService externalActorIdConfigurationService)
+            IExternalActorIdConfigurationService externalActorIdConfigurationService,
+            IUniqeMarketRoleGridAreaService uniqeMarketRoleGridAreaService)
```

Comment:
Fix spelling mistake

```suggestion
            IUniqueMarketRoleGridAreaService uniqueMarketRoleGridAreaService)
```